package.json

```diff
@@ -37,7 +37,6 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-minify": "^0.5.0",
     "enzyme": "^3.7.0",
-    "eslint": "5.6.0",
```

Comment:
did you remove eslint?

Reply:
I did remove eslint in the package.json . I was getting a CRA error similar (if not identical) to this issue https://github.com/facebook/create-react-app/issues/5247 so I removed it from package.json and nuked my node_modules and did an npm install and all was well after that. My apologies, I forgot I did that

Reply:
I'll add a .test.js today too

Reply:
That error that made me override the delete the eslint from package.json has resurfaced for some reason:

...........................
yarn start
yarn run v1.13.0
$ react-scripts start

There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "eslint": "5.12.0"

Don't try to install it manually: your package manager does it automatically.
However, a different version of eslint was detected higher up in the tree:

  /Users/gjoel/Documents/reactapps/react-transcript-editor/node_modules/eslint (version: 5.6.0)

Manually installing incompatible versions is known to cause hard-to-debug issues.

If you would prefer to ignore this check, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That will permanently disable this message but you might encounter other issues.

To fix the dependency tree, try following the steps below in the exact order:

  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "eslint" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.

In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:

  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.

  6. Check if /Users/gjoel/Documents/reactapps/react-transcript-editor/node_modules/eslint is outside your project directory.
     For example, you might have accidentally installed something in your home folder.

  7. Try running npm ls eslint in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed eslint.

If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.

P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
.........................................

Reply:
I'll add .env temporarily so I can at least test the .test.js file

Reply:
ok now added src/lib/Util/adapters/amazon-transcribe/index.test.js and pushed the changes. I did notice the test doesn't have a status of PASS but a status of RUNS and it doesn't exit. I'm not sure why that is though.
Do you need anything else from me for this PR @pietrop ?